packages/blocks/blocksAntd/src/blocks/MultipleSelector/MultipleSelector.js

```diff
@@ -103,6 +104,11 @@ const MultipleSelector = ({
                 methods.setValue(val);
                 methods.triggerEvent({ name: 'onChange' });
               }}
+              onSearch={async (inputVal) => {
+                setFetch(true);
+                await methods.triggerEvent({ name: 'onSearch', event: { inputVal } });
```

Comment:
@Gervwyk 
Maybe `searchValue` is a better name here?

Reply:
I think `onSearch` is more related to the specific event that is triggered. The fact that you search a value is related to the actions that you write for the event.